packages/fxa-shared/tsconfig.base.json

```diff
@@ -0,0 +1,13 @@
+{
+  "extends": "../../tsconfig.base.json",
+  "compilerOptions": {
+    "allowJs": true,
+    "declaration": true,
+    "emitDecoratorMetadata": true,
+    "experimentalDecorators": true,
+    "composite": true,
+    "types": ["mocha", "mozlog", "jest", "node"]
+  },
+  "include": ["./**/*", "./**/**/*", "./*", "./**/*.json"],
```

Comment:
Does that first include `["./**/*"]` not include the rest?

Reply:
hehe it came from https://github.com/mozilla/fxa/pull/15474/files#diff-0a0a6cc1daeea9323c25d6453a557da86b013b1b50019783dd9423848510c8e4L13 so I didn't look close at it.  I think you are right but I'm not sure because a) I don't know what tool or lib the patterns are passed to, and b) maybe there's a reason someone included the additional patterns?